src/classify.js

```diff
@@ -9,7 +9,7 @@ const witnessScriptHash = require('./templates/witnessscripthash')
 const witnessCommitment = require('./templates/witnesscommitment')
 
 const types = {
-  MULTISIG: 'multisig',
```

Comment:
This is a breaking change from `3.0.0`,  but,  we don't expose `classify` at this time (https://github.com/bitcoinjs/bitcoinjs-lib/issues/1148)